lnwire/features.go

```diff
@@ -139,6 +139,22 @@ const (
 	// sender-generated preimages according to BOLT XX.
 	AMPOptional FeatureBit = 31
 
+	// ExplicitChannelTypeOptional is a bit that denotes that a connection
+	// established with this node is to use explicit channel commitment
+	// types for negotiation instead of the existing implicit negotiation
+	// methods. With this bit, there is no longer a "default" implicit
+	// channel commitment type, allowing a connection to open/maintain types
+	// of several channels over its lifetime.
+	//
+	// TODO: Decide on actual feature bit value.
+	ExplicitChannelTypeOptional = 2020
```

Comment:
Not sure if this is important here (something something "it's okay to be odd"? maybe not relevant in this context), but all other flags have the required one even and the optional odd.

Reply:
Fixed. We still need to decide on an actual feature bit to be compatible with the other implementations. https://github.com/lightningnetwork/lightning-rfc/pull/880 so far doesn't mention anything about it.

Reply:
Yeah, I'm thinkign we can just pick a bit for now so we can use it on testnet, etc -- then after the spec PR is merged, we'll add that other bit before the next major release is cut.